decode.go

```diff
@@ -134,7 +154,10 @@ func UnmarshalTable(t *ast.Table, v interface{}) (err error) {
 		case *ast.Table:
 			fv, fieldName, found := findField(rv, key)
 			if !found {
-				return fmt.Errorf("line %d: field corresponding to `%s' is not defined in `%T'", av.Line, key, v)
+				if l := getLogger(); l != nil {
```

Comment:
Nothing we have is dependent on that behavior since everything migrated from `BurntSushi/toml`, so it's probably okay to change for our purposes. It really depends on how faithful we want this fork to remain.

That said, if we want to keep the original behavior, we could always log out to `ioutil.Discard` if you don't care.